crates/component/src/lib.rs

```diff
@@ -28,7 +28,10 @@ pub mod wasi {
     pub use super::bindings::wasi::*;
     pub use ::wasi::*;
 }
-pub use bindings::wasmcloud;
+
+pub mod wasmcloud {
+    pub use super::bindings::wasmcloud::*;
+}
```

Comment:
I'm a bit confused here, doesn't this achieve the same result as above?

Reply:
By moving the bus wit down a level, it changed the bindings generation. I liked keeping the bindings in this file the same with wasmcloud::bus vs bus